### PR TITLE
Add SID App Flavours

### PR DESCRIPTION
**Pull request by SebaMutuku**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

**Engineer Checklist**
- [ ] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [x] I have added any strings visible on UI components to the `strings.xml` file
- [ ] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [ ] I have built and run the FHIRCore app to verify my change fixes the issue and/or does not break the app 
- [ ] I have checked that this PR does NOT introduce **breaking changes** that require an update to **_Content_** and/or **_Configs_**? _If it does add a sample here or a link to exactly what changes need to be made to the content._

**Code Reviewer Checklist**
- [ ] I have verified **Unit tests** have been written for any new feature(s) and edge cases
- [ ] I have verified any strings visible on UI components are in the `strings.xml` file
- [ ] I have verifed the [CHANGELOG.md](./CHANGELOG.md) file has any notable changes to the codebase
- [ ] I have verified the solution has been implemented in a configurable and generic way for reuseable components
- [ ] I have built and run the FHIRCore app to verify the change fixes the issue and/or does not break the app

### CODE REVIEW

File: android/quest/build.gradle.kts [hunk 1]
```diff
@@ -240,6 +240,12 @@ android {
       versionNameSuffix = "-cadre"
       manifestPlaceholders["appLabel"] = "Cadre App"
     }
+    create("immunization") {
```
Comment (ellykits) [inline android/quest/build.gradle.kts:243]: Name after the project.

**Review by ellykits — CHANGES_REQUESTED** [review]

File: android/quest/build.gradle.kts (hunk 1 above)
Comment (SebaMutuku) [reply android/quest/build.gradle.kts:243]: Done

File: android/quest/build.gradle.kts [hunk 2]
```diff
@@ -240,6 +240,12 @@ android {
       versionNameSuffix = "-cadre"
       manifestPlaceholders["appLabel"] = "Cadre App"
     }
+    create("sideir") {
+      dimension = "apps"
+      applicationIdSuffix = ".sideir"
+      versionNameSuffix = "-sideir"
+      manifestPlaceholders["appLabel"] = "SID EIR"
+    }
```
Comment (pld) [inline android/quest/build.gradle.kts:248]: if we're prefixing this with `sid`, should we prefix the `cadre` and `bunda` app with sid?

thinking about it, the specificity here might be more to Indonesia vs SID, what about?

- `id-bunda`
- `id-cadre`
- `id-eir`

cc @MargaretNjenga @f-odhiambo

Comment (f-odhiambo) [reply android/quest/build.gradle.kts:248]: I agree to rename all of them.

**Review by ellykits — APPROVED** [review]
Document this naming convention for application flavors.

**Review by f-odhiambo — APPROVED** [review]

**Pull request merged**